src/System.Net.Http.WinHttpHandler/tests/UnitTests/FakeInterop.cs

```diff
@@ -388,6 +388,15 @@ private static bool CopyToBufferOrFailIfInsufficientBufferLength(string value, I
             return true;
         }
 
+        public static bool WinHttpQueryOption(
+            SafeWinHttpHandle handle,
+            uint option,
+            ref uint buffer,
+            ref uint bufferSize)
+        {
+            return true;
```

Comment:
The argument signatures for methods in FakeInterop.cs need to match the actual p/invoke signatures used in the product code and Interop.winhttp_types.cs file. Otherwise, it won't compile.

So, what are you suggesting should be changed in this FakeInterop.cs source file?